static/js/components/dashboard/CourseDescription.js

```diff
@@ -55,6 +56,32 @@ export default class CourseDescription extends React.Component {
     return '';
   };
 
+  renderViewCourseLink = (firstRun: CourseRun): React$Element<*>|void => {
```

Comment:
could you rename the `firstRun` parameter to just `courseRun`? It makes sense to call it `firstRun` down below, but this function really just takes a `courseRun` and renders something with it, so the `firstRun` name is a bit confusing.